mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -2626,6 +2626,9 @@ impl MarketCoinOps for TendermintCoin {
     #[inline]
     fn min_trading_vol(&self) -> MmNumber { self.min_tx_amount().into() }
 
+    #[inline]
+    fn should_burn_dex_fee(&self) -> bool { false }
```

Comment:
For tendermint, it should be split and sent directly to dex fee address and pre-burn address. Similar to how it's done for utxo. You can add this in another PR.

Reply:
I suggest add this in a different PR (as this one grows too big)